partcad/src/partcad/__init__.py

```
@@ -54,3 +56,19 @@
+    sentry_sdk.init(
+        dsn=user_config.get_string("sentry.dsn"),
+        release=__version__,
+        debug=user_config.get_bool("sentry.debug"),
+        shutdown_timeout=user_config.get_int("sentry.shutdown_timeout"),
+        enable_tracing=True,
+        attach_stacktrace=True,
+        traces_sample_rate=user_config.get_float("sentry.traces_sample_rate"),
+        integrations=[
+            LoggingIntegration(
+                level=logging.ERROR,
+            )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)